diem-unified-charter.md

```diff
@@ -12,47 +9,44 @@ ISO 7010 defines a set of emblems that can be used to identify hazards, these in
-The initial scope of the working group will assume that an emblem is also discovered through digital communication, however, ideally the methods defined for emblem creation and validation should not depend on this assumption.
+The DIEM WG will define an architecture and discovery mechanism enabling digital emblems to be presented and validated across applications and platforms in a cohesive way.
 
 # Architectural Consideration
 
-"To bear an emblem" means to present or display a symbol, figure, or device as an identifying mark. 
+"To issue or produce" an emblem, is to design the identifying mark and its associated attributes.
+The emblem issuer's role is to ensure that the identifying mark is easy to recognize and well understood.
+
+"To bear an emblem" means to present or display an identifying mark. 
 The entity that bears the emblem is respectively the bearer or emblem holder. 
 This can be a separate entity from the emblem issuer.
 
 "To validate an emblem" means to confirm the authenticity or legitimacy of a particular symbol or design, often by checking its details against a known standard or reference point. 
-Emblems may be observed by validators without the knowledge of the bearer displaying the emblem, or may be presented to a specific verifier upon request.
-Cryptographic verification can be optional as long is the emblem can be correctly interpreted.
+Emblems may be observed by validators without the knowledge of the bearer displaying the emblem, or may be presented to a specific validator upon request.
+Cryptographic verification can be optional as long as the emblem can be correctly interpreted.
 Therefore, to be effective, the semantics of an emblem must be well known, easily recognizable, and distinguishable from other emblems.
 
-Digital emblems can be bound to entities that are analog (e.g., a vehicle) or digital (e.g., a web page), however, there needs to be a digital representation that is used for validation using digital communication.
-
-Digital emblems can fall into multiple types (e.g., self-signed, attested, etc.) that will affect how a validator interprets and trusts the assertions made within the emblem.
-The communication between an entity that bears an emblem and a validator will influence requirements on how validators retrieve digital emblems. 
-These requirements will be key drivers in the development of a digital emblem architecture.
+Digital emblems can be borne or presented by entities that are analog (e.g., a vehicle) or digital (e.g., a web page), however, there needs to be a digital representation that is used for validation using digital communication.
 
-The architecture, specified by this group, must be flexible enough to support multiple serialization formats, securing mechanisms, discovery mechanisms, and use cases. 
+Digital emblems can be unsigned, self-signed, or signed by the emblem issuer.
```

Comment:
Attestation is a dangerous word : ) I hope this is clearer, it also related signing to the emblem issuer.

Reply:
fine to remove this sentence.